lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -143,9 +143,9 @@ <h3 class="hd hd-3 title">${_("Certificate Generation History")}</h3>
             <table>
                 <thead>
                 <tr>
-                    <td class="task-name"></td>
-                    <td class="task-date"></td>
-                    <td class="task-details"></td>
+                    <th class="task-name" scope="col">${_('Task name')}</th>
+                    <th class="task-date" scope="col">${_('Date')}</th>
```

Comment:
As I mentioned below, please provide a link to a course on the sandbox (with this code) where certificates are enabled so we can go through that workflow.